src/lldp_syncd/daemon.py

```diff
@@ -223,7 +223,7 @@ def parse_update(self, lldp_json):
             for interface in interface_list:
                 try:
                     # [{'if_name' : { attributes...}}, {'if_other': {...}}, ...]
-                    (if_name, if_attributes), = interface.items()
+                    (if_name, if_attributes), = list(interface.items())
```

Comment:
I believe this `list()` conversion is unnecessary. `2to3` errs on the side of caution and adds some conversions where they are not actually necessary.

Reply:
@jleveque Updated